src/utils/choice_random_items.js

```diff
@@ -0,0 +1,11 @@
+async function randomChoiceItems(length) {
+    const result = [];
+    while (result.length < 2) {
```

Comment:
1. I would prefer for instead of while (just IMHO its safer)
2. why this function is async?